crawl-ref/source/describe.cc

```diff
@@ -4347,6 +4368,7 @@ static string _monster_stat_description(const monster_info& mi)
     result << _monster_attacks_description(mi);
     result << _monster_missiles_description(mi);
     result << _monster_spells_description(mi);
+    _describe_monster_habitat(mi, result);
```

Comment:
Maybe it would be easier to make this function return a string; that way you can use the same `result << ` pattern. Might make your tests slightly cleaner too. What do you think?

Reply:
I was following the same patter as the functions:
`    _describe_monster_hp(mi, result);
    _describe_monster_ac(mi, result);
    _describe_monster_ev(mi, result);
    _describe_monster_mr(mi, result);`

I ended up moving that function call to a later part of describe because I thought it belonged somewhere else.

Which pattern works better? I'd be happy to switch it to a function that returns a string instead.

Reply:
I switched it to use a string, and renamed the function to be consistent with those other description functions. I think it's cleaner to do it this way. Let me know what you think!